### PR TITLE
Docs: Fixed broken links in Basic Concepts

### LINKED ISSUE #20877: Docs feedback: /guides/basic_concepts.md
> For more details on the user model for Grafana, please refer to Admin

`https://grafana.com/reference/admin` and  `https://grafana.com/reference/http_api#users` give `404`s.

### PR DESCRIPTION
**Pull request by oddlittlebird**
**What this PR does / why we need it**: Fixes a bunch of broken links in the Basic Concepts page, minor edits to an index page in datasources.

**Which issue(s) this PR fixes**:

Fixes #20877

**Special notes for your reviewer**: See question in comments. @robbymilo, please check my link syntax.

### CODE REVIEW

File: docs/sources/guides/basic_concepts.md [hunk 1]
```diff
@@ -4,117 +4,92 @@ description = "Grafana intro and concept guide"
...
+Panels like the [Graph]({{< relref "graph.md" >}}) panel allow you to graph as many metrics and series as you want. Other panels like [Singlestat]({{< relref "singlestat.md" >}}) require a reduction of a single query into a single number.
 
-Rows can be collapsed by clicking on the Row Title. If you save a Dashboard with a Row collapsed, it will save in that state and will not preload those graphs until the row is expanded.
+Panels can be made more dynamic with [Dashboard Templating]({{< relref "templating.md" >}}) variable strings within the panel configuration. The template can include queries to your data source configured in the Query Editor.
 
-### Panel
+Panels can be [shared]({{< relref "share_panel.md" >}}) easily in a variety of ways.
 
-The Panel is the basic visualization building block in Grafana. Each Panel provides a Query Editor (dependent on the Data Source selected in the panel) that allows you to extract the perfect visualization to display on the Panel by utilizing the Query Editor
+## Query editor
```
Comment (oddlittlebird) [inline docs/sources/guides/basic_concepts.md]: @torkelo , do we mean the Query Inspector?

File: docs/sources/guides/basic_concepts.md [hunk 2]
```diff
@@ -4,117 +4,92 @@ description = "Grafana intro and concept guide"
...
+Dashboards can be [shared]({{< relref "share_dashboard.md" >}}) easily in a variety of ways.
 
-Grafana supports multiple organizations in order to support a wide variety of deployment models, including using a single Grafana instance to provide service to multiple potentially untrusted Organizations.
+Dashboards can be tagged, and the dashboard picker provides quick, searchable access to all dashboards in a particular organization.
 
-In many cases, Grafana will be deployed with a single Organization.
+## Data source
 
-Each Organization can have one or more Data Sources.
+Grafana supports many storage backends for your time series *data source*. Each data source has a specific Query Editor that is customized for the features and capabilities that the particular data source exposes.
```
Comment (torkelo) [inline docs/sources/guides/basic_concepts.md]: Grafana can visualize, explore & alert on data from many different databases and cloud services. Each database or service type are accessed via a **data source**

File: docs/sources/guides/basic_concepts.md (hunk 1 above)
Comment (torkelo) [reply docs/sources/guides/basic_concepts.md]: no, I mean the query builder/editor that each data source has (looks different for each). This UI component builds the query.

File: docs/sources/guides/basic_concepts.md (hunk 2 above)
Comment (oddlittlebird) [reply docs/sources/guides/basic_concepts.md]: Updated.

**Comment by daniellee** [comment]
I tried testing this but I still got errors:

```
ERROR 2019/12/19 23:45:47 [en] REF_NOT_FOUND: Ref "annotations.md": "/site/content/guides/basic_concepts.md:25:34": failed to resolve ref: page reference
 "annotations.md" is ambiguous
WARN 2019/12/19 23:45:47 make non-relative ref/relref page reference(s) in page %q absolute, e.g. {{< ref "/blog/my-post.md" >}}
ERROR 2019/12/19 23:45:47 [en] REF_NOT_FOUND: Ref "annotations.md": "/site/content/guides/basic_concepts.md:25:34": failed to resolve ref: page reference
 "annotations.md" is ambiguous
```

**Comment by oddlittlebird** [comment]
@robbymilo or @DanCech , can you tell me what is wrong?

**Comment by robbymilo** [comment]
> I tried testing this but I still got errors:
> 
> ```
> ERROR 2019/12/19 23:45:47 [en] REF_NOT_FOUND: Ref "annotations.md": "/site/content/guides/basic_concepts.md:25:34": failed to resolve ref: page reference
>  "annotations.md" is ambiguous
> WARN 2019/12/19 23:45:47 make non-relative ref/relref page reference(s) in page %q absolute, e.g. {{< ref "/blog/my-post.md" >}}
> ERROR 2019/12/19 23:45:47 [en] REF_NOT_FOUND: Ref "annotations.md": "/site/content/guides/basic_concepts.md:25:34": failed to resolve ref: page reference
>  "annotations.md" is ambiguous
> ```



> @robbymilo or @DanCech , can you tell me what is wrong?

Those errors are Hugo saying it cannot find the page that was referenced in the shortcode. See my latest commit for the fix.

**Review by robbymilo — APPROVED** [review]

**Pull request merged**